source/unit/Unit.gd

```diff
@@ -143,6 +139,7 @@ func _set_experience_current(value: int) -> void:
 func _amla() -> void:
 	type.health += 3
 	type.experience *= 1.2
+	type.level += 1
```

Comment:
I am pretty sure the level does not increase on amla, only XP and Health. The Level is more representative for the advancement level of the unit.

Reply:
ye, I removed it